src/replica/prepare_list.h

```diff
@@ -67,8 +67,9 @@ class prepare_list : public mutation_cache, private replica_base
     // bulk load ingestion
     error_code prepare(mutation_ptr &mu,
                        partition_status::type status,
-                       bool pop_all_committed_mutations = false); // unordered prepare
-    virtual void commit(decree decree, commit_type ct);           // ordered commit
+                       bool pop_all_committed_mutations = false,
+                       bool secondary_commit = true);   // unordered prepare
```

Comment:
I recommend removing comment `// unordered prepare` and add comments for `secondary_commit` like `pop_all_committed_mutations`.

Reply:
Done.